python/hail/dataset.py

```diff
@@ -2745,6 +2745,52 @@ def linreg_multi_pheno(self, ys, covariates=[], root='va.linreg', use_dosages=Fa
                                            jarray(Env.jvm().java.lang.String, covariates), root, use_dosages, min_ac,
                                            min_af)
         return VariantDataset(self.hc, jvds)
+    
+    @handle_py4j
+    @requireTGenotype
+    @typecheck_method(ys=listof(strlike),
+                      covariates=listof(strlike),
+                      root=strlike,
+                      use_dosages=bool,
+                      variant_block_size=integral)
+    def linreg3(self, ys, covariates=[], root='va.linreg', use_dosages=False, variant_block_size=16):
```

Comment:
(this assumes performance bottleneck is in the linear algebra. i can see why you might want to keep as for initial ukbb where we'll prefilter variants once)